go/ai/gen.go

```diff
@@ -96,10 +96,12 @@ type GenerateResponse struct {
 	Usage      *GenerationUsage `json:"usage,omitempty"`
 }
 
+// A GenerateResponseChunk is the portion of the [GenerateResponse]
+// that is passed to a streaming callback.
 type GenerateResponseChunk struct {
 	Content []*Part `json:"content,omitempty"`
 	Custom  any     `json:"custom,omitempty"`
```

Comment:
Do you know the purpose of this field? Is it ever set in the JS?

Reply:
The Custom field? I don't think it is ever set in the JS.